planetarypraxis/smartforests
# Translate Home Page

Translating the home page requires:
  - Enable translation of tags (see https://docs.wagtail.org/en/stable/advanced_topics/i18n.html#translatable-snippets)
  - Update the tag cloud code to filter tags by locale
  - Update the sidebar code to only return pages in the current locale and display localized headings

These changes could potentially break things, so testing is required!

**To Test**

1. Make a copy of the live database and load it locally
2. Run the migrations
3. Create a translated version of a tag
4. Tag some localized pages with the localized tag
5. Check the homepage in that locale shows the localized tags and the correct pages in the sidebar

## How Will This Be Deployed?
We have to run the migrations.

## Screenshots (if appropriate):

## Types of changes

## Checklist:
- * I've checked the spec (e.g. Figma file) and documented any divergences.
- * My change requires a change to the documentation.
- * I've updated the documentation accordingly.
